Unity installed by unityhub crashes after opening projects

**Describe the bug**
If I open a project using unityhub it crashes soon after briefly displaying a loading screen.

**To Reproduce**
Steps to reproduce the behavior:
1. install any Unity version (I tried multiple 2019.3.* versions, 2018.4 and 2020.1b)
2. create a project (doesn't matter which template) or open an existing project 

**Expected behavior**
Running Unity

**Logs**
`~/.config/unity3d/Editor.log` (after creating/opening a project from the 3d template)
```
[LicensingClient] ERROR Failed to connect to local IPC
[Licensing::Module] Failed to connect to channel: LicenseClient-philm
[Licensing::Module] Successfully launched the LicensingClient
FailFast:
Couldn't find a valid ICU package installed on the system. Set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.

   at System.Environment.FailFast(System.String)
   at System.Globalization.GlobalizationMode.GetGlobalizationInvariantMode()
   at System.Globalization.GlobalizationMode..cctor()
   at System.Globalization.CultureData.CreateCultureWithInvariantData()
   at System.Globalization.CultureData.get_Invariant()
   at System.Globalization.CultureInfo..cctor()
   at System.StringComparer..cctor()
   at System.AppDomain.InitializeCompatibilityFlags()
   at System.AppDomain.Setup(System.Object)
[LicensingClient] Timed-out attempting to connect to the Licensing Client on pipe: LicenseClient-philm.
[LicensingClient] ERROR Failed to connect to local IPC
[Licensing::Module] Failed to connect to channel: LicenseClient-philm
IPC channel to LicensingClient doesn't exist; falling back to Legacy licensing
[LicensingClient] ERROR Failed to disconnect local IPC connection
Desktop is 7680 x 2160 @ 60 Hz

LICENSE SYSTEM [2020529 21:58:25] Next license update check is after 2020-05-30T18:33:28

[Package Manager] Server::Start -- Port 38910 was selected

 COMMAND LINE ARGUMENTS:
/home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Unity
-projectpath
/home/philm/dev/work/EDU/test-nixos
-useHub
-hubIPC
-cloudEnvironment
production
-hubSessionId
***
-accessToken
***
Successfully changed project path to: /home/philm/dev/work/EDU/test-nixos
/home/philm/dev/work/EDU/test-nixos
Using Asset Import Pipeline V2.
Adding genesis user token Adding genesis user token [Package Manager] Done resolving packages in 0.73s seconds
[Package Manager] Done checking package constraints in 0.00s seconds

(Unity:22206): Gtk-CRITICAL **: 21:58:28.798: gtk_container_foreach: assertion 'GTK_IS_CONTAINER (container)' failed
[Package Manager]
Registered 51 packages:
  Packages from [https://packages.unity.com]:
    com.unity.ads@3.4.4 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.ads@3.4.4)
    com.unity.analytics@3.3.5 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.analytics@3.3.5)
    com.unity.collab-proxy@1.2.16 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.collab-proxy@1.2.16)
    com.unity.ide.rider@1.1.4 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.ide.rider@1.1.4)
    com.unity.ide.vscode@1.1.4 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.ide.vscode@1.1.4)
    com.unity.multiplayer-hlapi@1.0.4 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.multiplayer-hlapi@1.0.4)
    com.unity.purchasing@2.0.6 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.purchasing@2.0.6)
    com.unity.render-pipelines.universal@7.3.1 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.render-pipelines.universal@7.3.1)
    com.unity.test-framework@1.1.13 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.test-framework@1.1.13)
    com.unity.textmeshpro@2.0.1 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.textmeshpro@2.0.1)
    com.unity.timeline@1.2.14 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.timeline@1.2.14)
    com.unity.xr.legacyinputhelpers@1.3.11 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.xr.legacyinputhelpers@1.3.11)
    nuget.mono-cecil@0.1.5-preview (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/nuget.mono-cecil@0.1.5-preview)
    com.unity.render-pipelines.core@7.3.1 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.render-pipelines.core@7.3.1)
    com.unity.shadergraph@7.3.1 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.shadergraph@7.3.1)
    com.unity.ext.nunit@1.0.0 (location: /home/philm/dev/work/EDU/test-nixos/Library/PackageCache/com.unity.ext.nunit@1.0.0)
  Built-in packages:
    com.unity.2d.sprite@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.2d.sprite)
    com.unity.2d.tilemap@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.2d.tilemap)
    com.unity.ugui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.ugui)
    com.unity.modules.ai@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.ai)
    com.unity.modules.androidjni@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.androidjni)
    com.unity.modules.animation@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.animation)
    com.unity.modules.assetbundle@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.assetbundle)
    com.unity.modules.audio@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.audio)
    com.unity.modules.cloth@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.cloth)
    com.unity.modules.director@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.director)
    com.unity.modules.imageconversion@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.imageconversion)
    com.unity.modules.imgui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.imgui)
    com.unity.modules.jsonserialize@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.jsonserialize)
    com.unity.modules.particlesystem@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.particlesystem)
    com.unity.modules.physics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.physics)
    com.unity.modules.physics2d@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.physics2d)
    com.unity.modules.screencapture@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.screencapture)
    com.unity.modules.terrain@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.terrain)
    com.unity.modules.terrainphysics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.terrainphysics)
    com.unity.modules.tilemap@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.tilemap)
    com.unity.modules.ui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.ui)
    com.unity.modules.uielements@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.uielements)
    com.unity.modules.umbra@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.umbra)
    com.unity.modules.unityanalytics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unityanalytics)
    com.unity.modules.unitywebrequest@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequest)
    com.unity.modules.unitywebrequestassetbundle@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestassetbundle)
    com.unity.modules.unitywebrequestaudio@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestaudio)
    com.unity.modules.unitywebrequesttexture@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequesttexture)
    com.unity.modules.unitywebrequestwww@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestwww)
    com.unity.modules.vehicles@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.vehicles)
    com.unity.modules.video@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.video)
    com.unity.modules.vr@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.vr)
    com.unity.modules.wind@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.wind)
    com.unity.modules.xr@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.xr)
    com.unity.modules.subsystems@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.subsystems)

[Subsystems] No new subsystems found in resolved package list.
[Package Manager] Done registering packages in 0.00s seconds
[Unity Package Manager (Upm)]
Parent process [22206] was terminated
```
`~/.config/unity3d/upm.log`
```
[2020-05-29T19:58:26.125Z][INFO] Starting Server
[2020-05-29T19:58:26.128Z][INFO] Server started on port [38910]
[2020-05-29T19:58:26.223Z][INFO] Health Request received
[2020-05-29T19:58:28.169Z][INFO] Resolving dependencies by depth
[2020-05-29T19:58:31.131Z][ERROR] [Unity Package Manager (Upm)]
Parent process [22206] was terminated
```
`~/.config/UnityHub/logs/info-log.json` (relevant parts)
```json
{"moduleName":"LocalProject","level":"info","message":"[ 'openProject projectPath: /home/philm/dev/work/EDU/test-nixos, current editor:',\n  undefined ]","timestamp":"2020-05-29T19:58:20.352Z"}
{"moduleName":"RestEvent:Project","level":"info","message":"[ 'onProjectOpen data:',\n  { projectPath: '/home/philm/dev/work/EDU/test-nixos' } ]","timestamp":"2020-05-29T19:58:20.352Z"}
{"moduleName":"UnityIPCServer","level":"info","message":"[ 'client process has connected to /tmp/Unity-hubIPCService.sock!' ]","timestamp":"2020-05-29T19:58:25.618Z"}
{"moduleName":"LaunchProcess","level":"info","message":"[ 'child process exited with code null' ]","timestamp":"2020-05-29T19:58:30.154Z"}
{"moduleName":"HubIPCService","level":"info","message":"[ 'socket has disconnected: [object Object] , false!' ]","timestamp":"2020-05-29T19:58:30.154Z"}
```

The logs from the 3d template based project also don't say very much interesting.
`Packages-Update.log` (the only log in `/Logs` of the project)
```

=== Fri May 29 22:22:57 2020

Packages were changed.
Update Mode: mergeDefaultDependencies

The following packages were added:
  com.unity.collab-proxy@1.2.16
  com.unity.ide.rider@1.1.4
  com.unity.ide.vscode@1.1.4
  com.unity.modules.ai@1.0.0
  com.unity.modules.androidjni@1.0.0
  com.unity.modules.animation@1.0.0
  com.unity.modules.assetbundle@1.0.0
  com.unity.modules.audio@1.0.0
  com.unity.modules.cloth@1.0.0
  com.unity.modules.director@1.0.0
  com.unity.modules.imageconversion@1.0.0
  com.unity.modules.imgui@1.0.0
  com.unity.modules.jsonserialize@1.0.0
  com.unity.modules.particlesystem@1.0.0
  com.unity.modules.physics@1.0.0
  com.unity.modules.physics2d@1.0.0
  com.unity.modules.screencapture@1.0.0
  com.unity.modules.terrain@1.0.0
  com.unity.modules.terrainphysics@1.0.0
  com.unity.modules.tilemap@1.0.0
  com.unity.modules.ui@1.0.0
  com.unity.modules.uielements@1.0.0
  com.unity.modules.umbra@1.0.0
  com.unity.modules.unityanalytics@1.0.0
  com.unity.modules.unitywebrequest@1.0.0
  com.unity.modules.unitywebrequestassetbundle@1.0.0
  com.unity.modules.unitywebrequestaudio@1.0.0
  com.unity.modules.unitywebrequesttexture@1.0.0
  com.unity.modules.unitywebrequestwww@1.0.0
  com.unity.modules.vehicles@1.0.0
  com.unity.modules.video@1.0.0
  com.unity.modules.vr@1.0.0
  com.unity.modules.wind@1.0.0
  com.unity.modules.xr@1.0.0
  com.unity.test-framework@1.1.13
  com.unity.textmeshpro@2.0.1
  com.unity.timeline@1.2.14
  com.unity.ugui@1.0.0
``` 
**Additional context**
It's kinda weird I had Unity running when I installed unityhub the first time (around 1 1/2 months ago)
But suddenly (a few days after that) it started crashing, soon after opening a project  (I guess it was a `nixos-rebuild --upgrade`) .

I think I invested now multiple hours to debug this issue, but I'm at the end of my abilities (as I'm also rather new to NixOS) since the error logs are rather sparse regarding this issue.

I deleted all kinds of side-effects of Unity Hub and Unity. At least I think that I deleted all of them: I searched for all kinds of unity variations in the home folder, so I hope I was thorough enough.

Fortunately I got it working with flatpak, but there I have now other issues (they have no git included which is needed for some UPM packages)
But the flatpak `Editor.log` (in which it's working) might be interesting for context:

`Editor.log` (I've ommitted most of the log, as it was very long, and in my opinion not relevant for this issue)
```
[LicensingClient] ERROR Failed to connect to local IPC
[Licensing::Module] Failed to connect to channel: LicenseClient-philm
[Licensing::Module] Successfully launched the LicensingClient
[LicensingClient] Timed-out attempting to connect to the Licensing Client on pipe: LicenseClient-philm.
[LicensingClient] ERROR Failed to connect to local IPC
[Licensing::Module] Failed to connect to channel: LicenseClient-philm
IPC channel to LicensingClient doesn't exist; falling back to Legacy licensing
[LicensingClient] ERROR Failed to disconnect local IPC connection
Desktop is 7680 x 2160 @ 60 Hz

LICENSE SYSTEM [2020529 22:37:54] Next license update check is after 2020-04-28T15:10:12

[Package Manager] Server::Start -- Port 59652 was selected

 COMMAND LINE ARGUMENTS:
/home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Unity
-projectpath
/home/philm/Downloads/test-nixos-lts
-useHub
-hubIPC
-cloudEnvironment
production
-buildTarget
Android
-hubSessionId
***
Successfully changed project path to: /home/philm/Downloads/test-nixos-lts
/home/philm/Downloads/test-nixos-lts
Using Asset Import Pipeline V2.
Rebuilding Library because the asset database could not be found!
Adding genesis user token Packages were changed.
Update Mode: updateDependencies

The following packages were added:
  com.unity.2d.sprite@1.0.0
  com.unity.2d.tilemap@1.0.0
  com.unity.ads@3.4.4
  com.unity.analytics@3.3.5
  com.unity.multiplayer-hlapi@1.0.4
  com.unity.purchasing@2.0.6
  com.unity.xr.legacyinputhelpers@1.3.11

(Filename: ./Modules/PackageManager/Editor/Requests/PackageManagerUpdateAsyncRequests.cpp Line: 155)

Adding genesis user token [Package Manager] Done resolving packages in 1.44s seconds
[Package Manager] Done checking package constraints in 0.00s seconds

(Unity:102): Gtk-CRITICAL **: 22:38:06.022: gtk_container_foreach: assertion 'GTK_IS_CONTAINER (container)' failed
[Package Manager]
Registered 48 packages:
  Packages from [https://packages.unity.com]:
    com.unity.ads@3.4.4 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.ads@3.4.4)
    com.unity.analytics@3.3.5 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.analytics@3.3.5)
    com.unity.collab-proxy@1.2.16 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.collab-proxy@1.2.16)
    com.unity.ide.rider@1.1.4 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.ide.rider@1.1.4)
    com.unity.ide.vscode@1.1.4 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.ide.vscode@1.1.4)
    com.unity.multiplayer-hlapi@1.0.4 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.multiplayer-hlapi@1.0.4)
    com.unity.purchasing@2.0.6 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.purchasing@2.0.6)
    com.unity.test-framework@1.1.13 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.test-framework@1.1.13)
    com.unity.textmeshpro@2.0.1 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.textmeshpro@2.0.1)
    com.unity.timeline@1.2.14 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.timeline@1.2.14)
    com.unity.xr.legacyinputhelpers@1.3.11 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.xr.legacyinputhelpers@1.3.11)
    nuget.mono-cecil@0.1.5-preview (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/nuget.mono-cecil@0.1.5-preview)
    com.unity.ext.nunit@1.0.0 (location: /home/philm/Downloads/test-nixos-lts/Library/PackageCache/com.unity.ext.nunit@1.0.0)
  Built-in packages:
    com.unity.2d.sprite@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.2d.sprite)
    com.unity.2d.tilemap@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.2d.tilemap)
    com.unity.ugui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.ugui)
    com.unity.modules.ai@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.ai)
    com.unity.modules.androidjni@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.androidjni)
    com.unity.modules.animation@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.animation)
    com.unity.modules.assetbundle@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.assetbundle)
    com.unity.modules.audio@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.audio)
    com.unity.modules.cloth@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.cloth)
    com.unity.modules.director@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.director)
    com.unity.modules.imageconversion@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.imageconversion)
    com.unity.modules.imgui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.imgui)
    com.unity.modules.jsonserialize@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.jsonserialize)
    com.unity.modules.particlesystem@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.particlesystem)
    com.unity.modules.physics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.physics)
    com.unity.modules.physics2d@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.physics2d)
    com.unity.modules.screencapture@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.screencapture)
    com.unity.modules.terrain@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.terrain)
    com.unity.modules.terrainphysics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.terrainphysics)
    com.unity.modules.tilemap@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.tilemap)
    com.unity.modules.ui@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.ui)
    com.unity.modules.uielements@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.uielements)
    com.unity.modules.umbra@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.umbra)
    com.unity.modules.unityanalytics@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unityanalytics)
    com.unity.modules.unitywebrequest@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequest)
    com.unity.modules.unitywebrequestassetbundle@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestassetbundle)
    com.unity.modules.unitywebrequestaudio@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestaudio)
    com.unity.modules.unitywebrequesttexture@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequesttexture)
    com.unity.modules.unitywebrequestwww@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.unitywebrequestwww)
    com.unity.modules.vehicles@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.vehicles)
    com.unity.modules.video@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.video)
    com.unity.modules.vr@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.vr)
    com.unity.modules.wind@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.wind)
    com.unity.modules.xr@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.xr)
    com.unity.modules.subsystems@1.0.0 (location: /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/PackageManager/BuiltInPackages/com.unity.modules.subsystems)

[Subsystems] No new subsystems found in resolved package list.
[Package Manager] Done registering packages in 0.00s seconds
Targeting platform: Android
Refreshing native plugins compatible for Editor in 0.00 ms, found 0 plugins.
Preloading 0 native plugins for Editor in 0.00 ms.
IsTimeToCheckForNewEditor: Update time 0 current 1590784687
Initialize engine version: 2019.3.11f1 (ceef2d848e70)
[Subsystems] Discovering subsystems at path /home/philm/Unity/Hub/Editor/2019.3.11f1/Editor/Data/Resources/UnitySubsystems
[Subsystems] Discovering subsystems at path /home/philm/Downloads/test-nixos-lts/Assets
...
```

Btw. while debugging I tried the recently new released Unity Hub version (2.3.2) with local nixpkgs. I already uploaded it to [archive.org](https://archive.org/details/unity-hub-2.3.2) with the hash:
`07nfyfp9apshqarc6pgshsczila6x4943hiyyizc55kp85aw0imn`
I also think that `git` should be in the `extraPkgs` as well (as mentioned for flatpak)

**Notify maintainers**
@tesq0
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.42, NixOS, 20.09pre227294.46f975f81e0 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.4`
 - channels(philm): `""`
 - channels(root): `"nixos-20.09pre227294.46f975f81e0"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/tools/unityhub/default.nix
+++ b/pkgs/development/tools/unityhub/default.nix
@@ -11,7 +11,10 @@ in appimageTools.wrapType2 rec {
     libpqxx gtk3 libsecret lsb-release openssl nodejs ncurses5
 
     libX11 libXcursor libXdamage libXfixes libXrender libXi
-    libXcomposite libXext libXrandr libXtst libSM libICE libxcb ]);
+    libXcomposite libXext libXrandr libXtst libSM libICE libxcb
+
+    libselinux pciutils libpulseaudio
+  ]);
 
   profile = ''
     export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS

